www/combobox.php

```diff
@@ -23,7 +23,7 @@
 function makeComboBox($name, $textlen, $curval, $vals, $onSet = "null")
 {
     // write the input field
-    $txt = "<input type=\"text\" name=\"$name\" id=\"$name\" "
+    $txt = "<span class='nobr'><input type=\"text\" name=\"$name\" id=\"$name\" "
```

Comment:
Now there's a mix of `"` and `'` in the generated HTML, in case it matters to you.

Reply:
doesn't matter to me 🤷